### PR TITLE
Documentation tidy

### PR DESCRIPTION
**Pull request by xross**
- Fix documentation build warnings
- Fix documentation html image rendering (pdf not supported)
- Tidy doc building in CI
- Rename examples as appropriate 
- General documentation tidies
- Removed separate example readmes and placed appropriate info in main doc

### CODE REVIEW

File: doc/rst/lib_board_support.rst [hunk 1]
```diff
@@ -110,15 +110,57 @@ XK_EVK_XU316 API
...
+They show how to use the cross-tile communications in conjunction with the |I2C| master server.
+The applications only setup the hardware and then exit the |I2C| server.
 
 XC Usage Example
 ================
 
-A more sophisticated example is provided, written in XC, by `app_xu316_i2s_loopback <https://github.com/xmos/lib_board_support/tree/develop/examples/app_xu316_i2s_loopback>`_. This application sets up the hardware on the `XK_AUDIO_316_MC_AB` board and then runs an |I2S| loopback application to take samples from the on-board ADCs and send the signals out through the DACs. It keeps the |I2C| master server running in case the application wants to change sample rate which requires |I2C| configuration of the audio CODECs.
+A more sophisticated example is provided, written in XC, by `app_xk_audio_316_mc_loopback`.
+
+This application sets up the hardware on the `XK_AUDIO_316_MC` board and then runs an |I2S|
```
Comment (danielpieczko) [inline doc/rst/lib_board_support.rst:130]:
```suggestion
This application sets up the hardware on the `XK-AUDIO-316-MC` board and then runs an |I2S|
```

File: doc/rst/lib_board_support.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ lib_board_support
 Introduction
 ************
 
-This repo contains board specific hardware configuration code for various ``XMOS`` evaluation and development kits. 
+This repo contains board specific hardware configuration code for various ``XMOS`` evaluation and development kits.
```
Comment (danielpieczko) [inline doc/rst/lib_board_support.rst:9]:
```suggestion
This repo contains board specific hardware configuration code for various `XMOS` evaluation and development kits.
```

**Review by danielpieczko — APPROVED** [review]
A couple of comments but otherwise all looks good.

**Comment by xross** [comment]
> A couple of comments but otherwise all looks good.

Thanks!

**Pull request merged**